### ISSUE TITLE
USB Port

### ISSUE THREAD

**Issue by ct1esj** [yl3im/no-pi-spot#1]
The project is really interesting, but I can't get it to work, it doesn't recognize the usb port.

Regards

CT1ESJ - Henrique

**Comment by yl3im** [comment]
What adaptor do you use to connect your MMDVM Hat to PC?
What does `journalctl -f `show right after you connect it?

**Comment by ct1esj** [comment]
Hello Inga.
First of all, I'm not using a native Linux machine, I'm experimenting with the Linux subsystem for Windows.
Then the hardware part is composed by an MMDVM_HS_HAT and the interface board for the usb port MMDVM_HS USB HAT, it is the first image that you put in the project. Well, as for the journalctl, only later at night when I get home can I see it, then I'll send a printscreen.
Thanks for your interest and feedback.

**Comment by yl3im** [comment]
Hi Henrique,

I didn't try with WSL as I only use Linux as my host system. I intend to implement a dockerized solution in near future, so it is truly system independent. The existing one is just a start, an MVP.

There also is an existing solution with Windows native MMDVM, by UA6AHP: https://www.youtube.com/watch?v=Bsv2L6HUxxA

**Comment by ct1esj** [comment]
Thanks Inga for the video link.
Hi! I didn't notice anything, but pictures are worth a thousand words. I'll try it out tonight. Your project is still very interesting, more than ever it makes perfect sense, at a time when there is a huge shortage of raspberrys, so, as I don't have any native machine for linux, for now I'm going to stay at interest in the project as I follow it. Thanks for everything and sorry for any inconvenience. Happy Christmas holidays.

**Comment by yl3im** [comment]
Also, have you followed the documented path for attaching USB devices to a WSL instance?

- https://learn.microsoft.com/en-us/windows/wsl/connect-usb
- https://blog.golioth.io/program-mcu-from-wsl2-with-usb-support/
- https://www.xda-developers.com/wsl-connect-usb-devices-windows-11/

I never tried it myself, just googled quickly :)

**Comment by ct1esj** [comment]
No, I haven't had time yet, the day should have 25 or 26 hours, hihihi. I'll see later
